.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 .env
-venv
\ No newline at end of file
+venv
+.env.local
```

Comment:
this should be public to run things locally.
If you want to add the Prod db you can create a .env.prod and then put it in the gitignore.